sklearn/linear_model/ridge.py

```diff
@@ -1048,6 +1048,16 @@ def _matmat(self, v):
         return res
 
 
+class _IdentityEstimator:
```

Comment:
We are starting to need those hacks everywhere... We talked about this with @ogrisel but can't remember where